erpnext/education/doctype/quiz/quiz.py

```diff
@@ -44,11 +45,34 @@ def evaluate(self, response_dict, quiz_name):
 			status = "Pass"
 		else:
 			status = "Fail"
+
+		if self.assessment_questions:
+			for question in response_dict.items():
```

Comment:
Is there a cleaner way to do this? Its a nested loop with a try catch block inside of it - just looks ugly